python/casa_distro/vbox.py

```diff
@@ -398,6 +399,12 @@ def copy_user(self, source, dest_dir):
                          'copyto', '--target-directory', dest_dir,
                          source])
 
+
+    def install_casa_distro(self, dest):
```

Comment:
We need to keep this method: it *is* called in `build_image.py` (for casa-run at the moment, but I have moved it to casa-dev and will push when that is tested). I will also maybe use it in `create_user_image`.

Reply:
I did changed the content.